### PR TITLE
Add examples for DD_APM_IGNORE_RESOURCES and DD_APM_ANALYZED_SPANS

### PR DESCRIPTION
**Pull request by cloud-orange**
### What does this PR do?
Adds config examples and supporting language for `DD_APM_IGNORE_RESOURCES` and `DD_APM_ANALYZED_SPANS` envvars. 

### Motivation
Docs specified intent of envvars but did not provide meaningful language as to the shape the configuration options could take or provide working examples

### Preview link

https://docs-staging.datadoghq.com/quotecenter/datadog-documentation/agent/docker/apm/?tab=java#docker-apm-environment-variables

### Additional Notes
Sourced info for `DD_APM_ANALYZED_SPANS` from https://docs.datadoghq.com/logs/log_collection/docker/?tab=containerinstallation#one-step-install-to-collect-all-the-container-logs and https://docs.datadoghq.com/agent/apm/?tab=agent630#trace-collection
Sourced info for `DD_APM_IGNORE_RESOURCES` from https://github.com/DataDog/datadog-agent/blob/master/pkg/config/config_template.yaml#L804

### CODE REVIEW

**Comment by cloud-orange** [comment]
I'm marking this a ready for review, BUT I'm not sure why the preview link isn't working.
Either I've misunderstood the URL convention or perhaps the service just takes a moment to spin up?

I will keep an eye on this for feedback.

**Comment by apigirl** [comment]
hey @cloud-orange - thanks for your PR! We are working on reviewing it. The preview link isn't working because you are working off of a fork. In order to use the preview links, you have to be an internal Datadog contributor. Don't worry though! We can pull it in locally to test your changes.

File: content/en/agent/docker/apm.md [hunk 1]
```diff
@@ -53,12 +53,14 @@ List of all environment variables available for tracing with the Docker Agent:
 | `DD_APM_DD_URL`            | Datadog API endpoint where traces are sent. For Datadog EU site set `DD_APM_DD_URL` to `https://trace.agent.datadoghq.eu` |
 | `DD_APM_RECEIVER_PORT`     | Port that the Datadog Agent's trace receiver listens on. Default value is `8126`.                                          |
 | `DD_APM_NON_LOCAL_TRAFFIC` | Allow non-local traffic when [tracing from other containers](#tracing-from-other-containers).                             |
-| `DD_APM_IGNORE_RESOURCES`   | A comma-separated list of endpoints and resources that the Agent should ignore (i.e. health checks endpoints).            |
-| `DD_APM_ANALYZED_SPANS`    | Configure the spans to analyze for transactions.                                                                          |
+| `DD_APM_IGNORE_RESOURCES`  | Configure resources for the agent to ignore. Format should be comma seperated\* regular expressions. i.e. `"GET /ignore-me,(GET|POST) /and-also-me"` |
```
Comment (apigirl) [inline content/en/agent/docker/apm.md]:
```suggestion
| `DD_APM_IGNORE_RESOURCES`  | Configure resources for the agent to ignore. Format should be comma seperated, regular expressions. i.e. <code>"GET /ignore-me,(GET&#124;POST) /and-also-me"</code>. |
```

File: content/en/agent/docker/apm.md [hunk 2]
```diff
@@ -53,12 +53,14 @@ List of all environment variables available for tracing with the Docker Agent:
...
+| `DD_APM_ANALYZED_SPANS`    | Configure the spans to analyze for transactions. Format should be comma seperated\* instances of `<SERVICE_NAME>|<OPERATION_NAME>=1`. i.e. `my-express-app|express.request=1,my-dotnet-app|aspnet_core_mvc.request=1` |
```
Comment (apigirl) [inline content/en/agent/docker/apm.md]:
```suggestion
| `DD_APM_ANALYZED_SPANS`    | Configure the spans to analyze for transactions. Format should be comma seperated instances of <code>\<SERVICE_NAME>&#124;\<OPERATION_NAME>=1</code>. i.e. <code>my-express-app&#124;express.request=1,my-dotnet-app&#124;aspnet_core_mvc.request=1</code>. |
```

File: content/en/agent/docker/apm.md [hunk 3]
```diff
@@ -53,12 +53,14 @@ List of all environment variables available for tracing with the Docker Agent:
...
 | `DD_APM_ENV`               | Sets the default [environment][2] for your traces.                                                                        |
 | `DD_APM_MAX_EPS`           | Sets the maximum APM events per second.                                                                                   |
 | `DD_APM_MAX_TPS`           | Sets the maximum traces per second.                                                                                       |
 
+**\****No additional characters such as spaces or newlines are allowed between commas*
```
Comment (apigirl) [inline content/en/agent/docker/apm.md]:
```suggestion
**Note**: No additional characters such as spaces or newlines are allowed between commas
```

Comment (cloud-orange) [reply content/en/agent/docker/apm.md]: I think you handled this better with the info block in your PR which is now merged. I think I'll just scrap this

**Comment by cloud-orange** [comment]
@apigirl So it seems like I've messed up the work flow by forking the repo first.

Perhaps I misunderstood the directions here: https://github.com/DataDog/documentation#outside-contributors specifically step one "fork the master branch"

**Comment by apigirl** [comment]
@cloud-orange - no, you didn't mess up the workflow. I think it would be more accurate for us to update the readme/PR template to explain that folks who are outside of the Datadog org can't use the the staging build

**Comment by apigirl** [comment]
thanks for the offer to update the readme! I think we're going to dig into this a little bit more and then update the PR template and the outside contributors doc to clarify

**Pull request merged**